src/System.Private.ServiceModel/tests/Scenarios/Extensibility/WebSockets/WebSocketTests.cs

```diff
@@ -169,10 +170,10 @@ public static void WebSocket_Http_Duplex_BinaryStreamed()
         }
     }
 
-    [Fact]
+    [ConditionalFact(nameof(WebSockets_Supported), nameof(Root_Certificate_Installed))]
     [OuterLoop]
 #if !FEATURE_NETNATIVE
-    [ActiveIssue(420, PlatformID.AnyUnix)]
+    //[ActiveIssue(420, PlatformID.AnyUnix)]
```

Comment:
I've decided to remove from this PR all attempts to remove or replace [ActiveIssue].  Once we close on #1034, we can apply those decisions.